src/test/kotlin/io/github/alexbroadbent/tsgen/TimestampFormatSpec.kt

```diff
@@ -15,6 +16,7 @@ class TimestampFormatSpec : ShouldSpec() {
     private val generator = TimestampGenerator
 
     init {
+        TimeZone.setDefault(TimeZone.getTimeZone("GMT+7:00"))
```

Comment:
It would be great to add a test for this rather than change the existing ones so we know that we preserve both zones.